site/js/SessionActor.js

```diff
@@ -49,7 +49,17 @@ export default class SessionActor extends Actor {
     let serverListOpen = false
     document.querySelector('.server-list-heading').addEventListener('click', async evt => {
       const serverListEl = document.getElementById('server-list')
-      serverListEl.style.setProperty('--server-list-height', ((Object.keys(this.sessionIDs).length + 1) * 54 + 32) + 'px')
+
+      // Set CSS variable used for animation. Numbers here relate to
+      // the heights of different elements in #server-list.
+      //
+      // This must be arbitrarilly set based on the number of servers
+      // we should display in the list.
+      //
+      // Using variables for this is *significantly* better than the
+      // horrible, unreliable max-height: auto hack for animating height.
+      serverListEl.style.setProperty('--server-list-height',
+        ((Object.keys(this.sessionIDs).length + 1) * 54 + 32) + 'px')
```

Comment:
Looks like this PR includes the changes in a commit from #52 (29e8709fd0330dcf2a3486dd714bab9c23303cd2)? This should probably be magick'd out so that you only have what was changed in side/index.html.

Reply:
oh man

Reply:
resolved, pls to approve